fem/gridfunc.hpp

```diff
@@ -633,6 +633,11 @@ class QuadratureFunction : public Vector
     */
    inline void GetElementValues(int idx, Vector &values) const;
 
+   /// Return the quadrature function values at an integration point.
+   /** The result is stored in the Vector @a values as a reference to the
+       global values. */
+   inline void GetElementValues(int idx, const int ip_num, Vector &values);
```

Comment:
Requested changes:
- Create a `const` version of this function that copies the data, see line 634.
- This will allow you to have a `const` reference to `QuadratureFunction` in `QuadratureFunctionCoefficient.`

Reply:
Now been added